packages/pigeon/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Pigeon Contributor's Guide
+
+**author:** gaaclarke\
+**updated:** 4/6/21
```

Comment:
Why is there an author and update date, given that both are going to be accurately captured in the git history? This seems likely to become stale.

Reply:
Yea, fair enough, removed.